Packs/CommonScripts/Scripts/SetGridField/README.md

```diff
@@ -78,5 +78,14 @@ keys="name, value"
 Grid after update: \
 ![Grid](https://github.com/demisto/content/raw/4510eafaf6cfeb48a42d9032dd0e71200b288ad5/Packs/Legacy/Scripts/SetGridField/doc_files/grid_list_update.png) 
 
+## Troubleshooting
+
+The first time you run `SetGridField` on a newly created grid field, you may see an error similar to the following:
+
+![Screen Shot 2021-12-21 at 10 36 03 PM](https://user-images.githubusercontent.com/91506078/147041267-2cb02f82-3bae-42ba-a15f-d8fceb44f44c.png)
```

Comment:
Better to use a relative path with the docs path
Packs/CommonScripts/Scripts/SetGridField/doc_files/
Move your file to the folder and in the readme use this path
![Screenshot name](../doc_files/screenshotfile.png)